lte/gateway/python/magma/pipelined/qos/common.py

```diff
@@ -104,26 +105,33 @@ def find_session_with_rule(self, rule_num: int) -> SubscriberSession:
                 break
         return session_with_rule
 
-    def update_rule(self, ip_addr: str, rule_num: int, d: FlowMatch.Direction,
-                    qos_handle: int) -> None:
-        k = get_subscriber_key(self.imsi, ip_addr, rule_num, d)
-        self._redis_store[get_json(k)] = qos_handle
+    def _update_rules_map(self, ip_addr: str, rule_num: int, d: FlowMatch.Direction,
+                    qos_data) -> None:
 
         if rule_num not in self.rules:
             self.rules[rule_num] = []
 
         session = self.get_or_create_session(ip_addr)
         session.rules.add(rule_num)
-        self.rules[rule_num].append((d, qos_handle))
```

Comment:
should the name be a bit more descriptive, rules is confusing because we're accessing it by rule_num to retrieve what exactly?

Reply:
I agree, But I would do it as followup patch to avoid complicating review.